openassessment/xblock/resolve_dates.py

```diff
@@ -46,7 +58,11 @@ def _parse_date(value, _):
         try:
             return parse_date(value).replace(tzinfo=pytz.utc)
```

Comment:
I don't think it's necessary. That's only used to compare dates to each other, so i just keep all of those in UTC and convert to timezone only when displaying to user.